ferretdb/ferretdb_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ferretdb
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+// ExampleNew is a testable example for Run func.
+func ExampleNew() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
Why we need a timeout there?

Reply:
> Why do we need a timeout there?

I'm still trying to figure out how it's better how-to run `fdb.Run`.
If I call it with the context with cancel, the re-run of the test gives an error that the port is busy.

Reply:
> > Why do we need a timeout there?
> 
> I'm still trying to figure out how it's better to run `fdb.Run`. If I call it with the context with cancel, the re-run of the test gives an error that the port is busy.

Reply:
removed timeout, changed test.